rendered/manifests/agent-only/clusterRole.yaml

```diff
@@ -1,83 +0,0 @@
----
```

Comment:
Why did you remove all the rendered files?

Reply:
added them back. `make render` only deleted them, but not create new ones.